### DISCUSSION TITLE
Example code after updating the new version

### DISCUSSION THREAD

**Discussion by lucasliunju** [hpcaitech/ColossalAI#150] [Community | General]
Hi,

I find some function names do not work in the examples after updating the new version.

For example, ColossalAI/examples/vit_b16_imagenet_data_parallel/train.py +80:

`optimizer = colossalai.nn.Lamb(model.parameters(), lr=1.8e-2, weight_decay=0.1)`

I guess it should be :

`optimizer = colossalai.nn.optimizer.Lamb(model.parameters(), lr=1.8e-2, weight_decay=0.1)`

**Comment by FrankLeeeee** [comment]
Hi, @lucasliunju . Let me test this and get back to you :)

**Comment by ver217** [comment]
Hi, @lucasliunju . I've updated the example and add more details in `README`. And as for your issue, I ran this example again and found there is no problem. I also checked the source code, and I'm sure that we import all optimizers in `colossalai/nn/__init__.py`. You can try it again :)
